src/CommandingOfficers/Commander.java

```diff
@@ -290,15 +295,15 @@ public ArrayList<GameOverlay> getMyOverlays(GameMap gameMap, boolean amIViewing)
   }
 
   // Note: Cart charge only uses whole HP, so that's what we're doing, too.
-  public double calculateCombatCharge(UnitDelta minion, UnitDelta enemy)
+  public int calculateCombatCharge(UnitDelta minion, UnitDelta enemy, boolean isCounter)
```

Comment:
Why add parameter and not use?

Reply:
Never mind. Found it in `RuinedCommander`. So only when the attacker is in the CO zone do we get charge.

Reply:
Correct.
I'm considering a game option to make it work as you'd expect (i.e. when *your* unit is in the zone), but also that sounds like effort.